### PR TITLE
Date visited appears in the reviews now.

### PR DESCRIPTION
**Pull request by Spellbreeze**
Contributor(s): Parkaranjot, Madhura, Xander

### CODE REVIEW

File: Civility Optics/Views/VenueDetails.swift [hunk 1]
```diff
@@ -77,6 +77,16 @@ var email: String
...
             VStack {
                 HStack(spacing: 4) {
+                    // Convert Date to String
+                    // Create Date Formatter
+                    let endOfSentence = result.date_visited.firstIndex(of: "T")!
+                    let date = result.date_visited[...endOfSentence]
+                    Text(date)
+                    Spacer()
+                }
+                HStack(spacing: 4) {
```
Comment (menelik-g) [inline Civility Optics/Views/VenueDetails.swift:87]: Good formatting for date

File: Civility Optics/Views/VenueDetails.swift [hunk 2]
```diff
@@ -77,6 +77,16 @@ var email: String
...
+                    // Convert Date to String
+                    // Create Date Formatter
                     Text("Rating: ")
                         .bold()
 //                        .multilineTextAlignment(.leading)
```
Comment (menelik-g) [inline Civility Optics/Views/VenueDetails.swift:92]: Line can be erased since its not doing anything

**Review by menelik-g — APPROVED** [review]
Good code, some commented code can be removed

File: Civility Optics/Views/VenueDetails.swift [hunk 3]
```diff
@@ -77,6 +77,16 @@ var email: String
           HStack {
             VStack {
                 HStack(spacing: 4) {
+                    // Convert Date to String
+                    // Create Date Formatter
+                    let endOfSentence = result.date_visited.firstIndex(of: "T")!
+                    let date = result.date_visited[...endOfSentence]
```
Comment (parkaransingh) [inline Civility Optics/Views/VenueDetails.swift:83]: Next commit should have the caption font.

**Review by parkaransingh — APPROVED** [review]
Good. add the name as well in your next commit!

**Pull request merged**